src/python/grpcio_tests/tests_aio/unit/init_test.py

```diff
@@ -32,6 +34,22 @@ def test_insecure_channel(self):
         self.loop.run_until_complete(coro())
 
 
+class TestSecureChannel(AioTestBase):
+    """Test a secure channel connected to a secure server"""
+
+    def test_secure_channel(self):
+
+        async def coro():
+            server_target, _ = await start_test_server(secure=True)  # pylint: disable=unused-variable
+            credentials = grpc.local_channel_credentials(
+                grpc.LocalConnectionType.LOCAL_TCP)
+            secure_channel = aio.secure_channel(server_target, credentials)
+
+            self.assertIsInstance(secure_channel, aio.Channel)
```

Comment:
Can we make sure that we're able to send a message?

Reply:
Wouldn't that be tested in ``test_call_credentials``? Please correct me if I'm wrong, or misunderstood.

Reply:
We do. But that's a call credential, whereas this is a channel credential. There are different code paths involved.